content/learn/publication-opportunities.md

```diff
@@ -0,0 +1,124 @@
+---
+Title: Pre- & Post-Publication Opportunities
+weight: 6400
+type: essay
+abstract:  "Funding, conferences, awards, and more for your Quire project"
+---
+
+## Pre-Publication
+
+### 💰 Funding 💰
+*Apply for grants, awards, and other funding before getting started*
+
+|Shortform|Granting Body|Description|
+|--|--|--|
+|ARiAH|Association of Research Institutes in Art History|[Digital Development Award for Art History](http://www.ariah.info/ARIAHAWARDS.html)|
+|CAA|College Art Association|[Publishing Grants](https://www.collegeart.org/programs/publishing-grants)|
+|IMLS|Institute of Museum and Library Services|[Digital Humanities Advancement Grants](https://www.imls.gov/grants/available/digital-humanities-advancement-grants)|
+|Knight|John S. and James L. Knight Foundation|[Arts Funding](https://knightfoundation.org/apply/)|
+|Kress|The Kress Foundation|[Digital Art History Grants](https://www.kressfoundation.org/Programs/Grants/Digital-Art-History)|
+|Mellon|The Andrew W. Mellon Foundation |[Grants](https://mellon.org/grants/grantmaking-policies-and-guidelines/grant-proposal-guidelines/)|
+|NEH|National Endowment for the Humanities|[Digital Humanities Advancement Grants](https://www.neh.gov/grants/odh/digital-humanities-advancement-grants)|
+
+## Post-Publication
+
+### 🎭 Showcase 🎭
```

Comment:
Rather than the masks, I would suggest the praise or "raising hands" emoji: https://emojipedia.org/raising-hands/. That's what we use on the forum to indicate Show & Tell. If that doesn't feel right to you, I'd also suggest the star or the flashlight.

Reply:
Ok, changed it to the raising hands emoji. Would it make more sense for this section to be called "Show & Tell" if "showcase" is the name of the Quire-specific showcase platform?

Reply:
ok, this change is reflected in the most recent commit